ChartboostAdapter/build.gradle.kts

```diff
@@ -85,15 +85,15 @@ android {
 
 dependencies {
     "localImplementation"(project(":ChartboostMediation"))
+    "localImplementation"(project(":ChartboostMonetization"))
```

Comment:
this breaks public builds though...
maybe we do this for now for the release then fix it later